### PR TITLE
Update README and command description

### PR DESCRIPTION
**Pull request by EstebanRomero84**
### What does this PR do?
Updates README and command description.
### What issues does this PR fix or reference?

[Insert GUS WI](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000018pwRAYAY/view)

### Functionality Before

README does not mention report and resume commands. 

### Functionality After

README mentions report and resume commands.

### CODE REVIEW

File: messages/deploy.pipeline.resume.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Resume watching a pipeline deploy operation.
 
 # description
 
-Use this command to resume watching a pipeline deploy operation if the original command times out or you specified the --async flag. Deploy pipeline operations include standard deployments, quick deployments and deployment validations.
+Use this command to resume watching a pipeline deploy operation if the original command times out or you specified the --async flag.
```
Comment (EstebanRomero84) [inline messages/deploy.pipeline.resume.md:7]: This release will not include quick deployment and validate deployment.

File: README.md [hunk 2]
```diff
@@ -145,3 +148,98 @@ FLAG DESCRIPTIONS
...
+
+```
+  USAGE
+    $ sf deploy pipeline resume -c <value> [--json] [-i <value>] [-r] [-w <value>]
+
+  FLAGS
+    -c, --devops-center-username=<value>  (required) [default: test-cubxekhq4e2c@example.com] Username or alias of the DevOps Center org.
+    -i, --job-id=<value>                  Job ID of the pipeline deploy operation you want to resume.
+    -r, --use-most-recent                 Use the job ID of the most recent deploy operation.
+    -w, --wait=<minutes>                  [default: 33 minutes] Number of minutes to wait for command to complete and display results.
```
Comment (npiccolo) [inline README.md]: I will update the flags here for verbose/concise during this story[W-12695207](https://gus.lightning.force.com/a07EE00001NG4OJYA1)

**Review by npiccolo — COMMENTED** [review]
Looks good to me !!!
I would wait for @ekapner review just to be sure we are writing all the right things =D

File: README.md (hunk 2 above)
Comment (ekapner) [reply README.md]: @EstebanRomero84 - we don't use `[default: test-cubxekhq4e2c@example.com]` in other -c descriptions and the username will be different for every org - so seems like saying a specific one is the default is misleading.

Comment (EstebanRomero84) [reply README.md]: You are right Emily, that should not be here. I already removed it.
Thanks.

**Review by npiccolo — APPROVED** [review]

**Pull request merged**